packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/XDD/Extraction.java

```diff
@@ -0,0 +1,79 @@
+package software.uncharted.terarium.hmiserver.models.xdd;
+
+import java.util.Date;
+import java.util.Hashtable;
+
+import javax.json.bind.annotation.JsonbProperty;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+/**
+ * XDD Document extraction representation
+ */
+@JsonFormat(with = JsonFormat.Feature.ACCEPT_CASE_INSENSITIVE_PROPERTIES)
+public class Extraction {
+
+	@JsonbProperty("ASKEM_CLASS")
+	public String ASKEM_CLASS;
+
+	@JsonbProperty("properties")
+  public ExtractionProperties properties;
+
+	@JsonbProperty("askem_id")
+	public String askem_id;
+
+	@JsonbProperty("_xdd_created")
+  public Date _xdd_created;
+
+	@JsonbProperty("_xdd_registrant")
+  public Number _xdd_registrant;
+}
+
+class ExtractionProperties {
```

Comment:
can you move this to a new file

Reply:
I would rather keep them in the same Extraction file for readability